packages/omg-js-childchain/src/rpc/rpcApi.js

```diff
@@ -13,26 +13,34 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
-const request = require('request-promise-native')
+const axios = require('axios')
 const debug = require('debug')('omg.childchain.rpc')
 const JSONBigNumber = require('omg-json-bigint')
-
+const HttpsProxyAgent = require('https-proxy-agent')
 class RpcError extends Error {
   constructor ({ code, description, messages }) {
     super(description || code + (messages ? `, ${messages.code}` : ''))
     this.code = code
   }
 }
+function getTransformResponse () {
```

Comment:
What's this for? Is it to override the default transformResponse?
Maybe add a comment explaining it

Reply:
yeah because it is using `JSON.parse` by default which will break big number